src/messaging/classes/Message.ts

```diff
@@ -128,12 +217,6 @@ export class Message extends HttpClient {
                     },
                 });
             }
-
-            let apiResource: string = '/v2/messages/sms';
-            if (message?.multimedia) {
-                apiResource = '/v2/messages/mms';
-            }
-
             const result = await this.instance.post<TMessageSendResponse>(
                 apiResource,
                 message
```

Comment:
@AnanditaBhatt-tdev the message payload will need to be mutated to support the Messaging API v2 service as the attr multimedia is not supported by the service and will throw an error due to issing attrs MMSContent & filename (lowercase).

May be a good idea to have separate v2 & v3 message types / interfaces, what do think?

Reply:
Makes sense. Will update that.